Gems/Camera/Code/Source/CameraComponentController.cpp

```diff
@@ -506,23 +552,24 @@ namespace Camera
 
     void CameraComponentController::OnViewportDefaultViewChanged(AZ::RPI::ViewPtr view)
     {
-        m_isActiveView = m_atomCamera == view;
+        m_isActiveView = m_atomCameraView->GetView(AZ::RPI::ViewType::Default) == view;
```

Comment:
this one could use its own interface to simplify it:

````
m_isActiveView = GetView() == view;
````

If there are other places in this class calling `m_atomCameraView->GetView(AZ::RPI::ViewType::Default)` directly, it'd be great to replace them with `GetView()`